tests/test_optics.py

```diff
@@ -53,3 +53,57 @@ def test_sunrise_sunset_fail():
 
     with pytest.raises(ValueError):
         sunrise, sunset = gt.optics.sunset_sunrise(time, lat, lon)
+
+
+@pytest.mark.parametrize(percentile, [95, 85, 50])
```

Comment:
I am so sorry this should be `@pytest.mark.parametrize('percentile', [95, 85, 50])` 🤦‍♂️. Sorry for the confusion...

Reply:
Thanks :) haha, sorry about all these commits. I'll get there...